gui/default/syncthing/core/syncthingController.js

```diff
@@ -1532,6 +1532,12 @@ angular.module('syncthing.core')
             delete $scope.folderRejections[folder + "-" + device];
         };
 
+        $scope.ignoreRejectedFolder = function (folder, device) {
+            $scope.config.ignoredFolders.push(folder.id);
```

Comment:
Ok well either that did it OR... just a rebuild did.  Don't know which but it's working now.  :D